packages/app-store/ee/routing-forms/lib/getSerializableForm.ts

```diff
@@ -17,7 +23,11 @@ export function getSerializableForm<TForm extends App_RoutingForms_Form>(form: T
   // Ideally we shouldb't have needed to explicitly type it but due to some reason it's not working reliably with VSCode TypeCheck
   const serializableForm: SerializableForm<TForm> = {
     ...form,
-    fields: fieldsParsed.data,
+    fields: fieldsParsed.data
```

Comment:
Centrally support not sending soft deleted fields. Fields can only be soft deleted.